Meetups/26/2025-03-27-minutes.md

```diff
@@ -0,0 +1,186 @@
+# Meeting Minutes
+
+## Info
+
+:date: **Date:** 27 March 2025
+
+### :bust_in_silhouette: Participants
+
+<!-- This list will copied over from the meeting tool -->
+- Ege Korkan
+- Cristiano Aguzzi
+- Henk Spaaj
+- Kunihiko Toumura
+- Jase Kraft
+- Tomoaki Mizushima
+- Sibla, Wilfried
+- Tamoghna Ojha
+- Piotr Sowiński
+- Yusuf Koç
+
+:writing_hand: **Scribe:** Ege
+
+----
+
+### Introduction
+
+- Cris: Welcome Everyone!
+
+### News
+
+Cris: No new meetup planned. You can let us know if you want to present something.
+
+Cris: We have the tutorial shaping up. Also 3 projects for the Google Summer of Code are available from Eclipse Thingweb.
+
+Cris: Joining the groups is free, you can join using the button.
+
+### Meetup Presentation
+
+Cris: We have Henk Spaaj here. He is a retired Senior Software Engineer and development manager. He was educated as Electronics Engineer and has over 30 years of software development experience in embedded industrial and consumer solutions
+
+Henk: I did development before for enterprises etc. Since 2023 doing WoT.
+
+Henk: I bought a piece of land had pump house built a pump house on it and the one you see in the picture. The house itself is a bit further away and if something goes something goes wrong in that pump house then, I'd rather be ahead of the game than finding out that the water is frozen.
+
+Henk: I have started with MQTT and myzone. Then went to NGSI-LD.
```

Comment:
This line can be removed. See comment on the previous paragraphs.

Reply:
```suggestion
```